commit d3a319ceb64dd7c32bf5e2955ea60e18053b80e7
Author: Florian Westphal <fw@strlen.de>
Date:   Mon Jan 16 18:24:56 2017 +0100

    netfilter: conntrack: remove GC_MAX_EVICTS break
    
    commit 524b698db06b9b6da7192e749f637904e2f62d7b upstream.
    
    Instead of breaking loop and instant resched, don't bother checking
    this in first place (the loop calls cond_resched for every bucket anyway).
    
    Suggested-by: Nicolas Dichtel <nicolas.dichtel@6wind.com>
    Signed-off-by: Florian Westphal <fw@strlen.de>
    Acked-by: Nicolas Dichtel <nicolas.dichtel@6wind.com>
    Signed-off-by: Pablo Neira Ayuso <pablo@netfilter.org>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1360_d3a319ceb64dd7c32bf5e2955ea60e18053b80e7.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1360_d3a319ceb64dd7c32bf5e2955ea60e18053b80e7.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
